docs/team/kayhian.md

```diff
@@ -9,38 +9,32 @@ ResearchRoster is a desktop address book application used for teaching Software
 
 Given below are my contributions to the project.
 
-* **New Feature**: Added the ability to undo/redo previous commands.
-  * What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
-  * Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-  * Highlights: This enhancement affects existing commands and commands to be added in the future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+* **New Feature**: Added `assign` command to (randomly) assign persons to study group(s) [\#125]()
+  * What it does: allows the user to randomly assign a list of persons into study groups.
+  * Justification: As our product is targeted at researchers, we anticipate the need to quickly and fairly assign participants to study groups. This feature allows the user to assign an entire list of participants in one command call.
+  * Highlight: a bonus feature of this command is the ability to assign all persons in the displayed list to a single study group by inputting only one staudy group name.
```

Comment:
study is spelt as staudy :(

Reply:
Thank you. I have fixed the bugs mentioned.